src/ansi-c/expr2c.cpp

```diff
@@ -2035,7 +2035,8 @@ std::string expr2ct::convert_struct(
 /// To generate a C-like string representing a struct. Can optionally include
 /// the padding parameters.
 /// \param src: The struct declaration expression
-/// precedence
+/// \param [out] precedence: Precedence of the top level operator in the
+///   resulting string, used to decide about adding parenthesesis
```

Comment:
parenthesesis -> parentheses. [Parenthesis is also a word, as I just learned](https://en.oxforddictionaries.com/definition/parenthesis)

Reply:
👍